### PR TITLE
typos in loops.tex

### PR DESCRIPTION
**Pull request by bryangingechen**

### CODE REVIEW

File: blueprint/src/loops.tex [hunk 1]
```diff
@@ -222,9 +222,9 @@ \subsection{Surrounding families}
 The difficulty in constructing global families of surrounding loops is that
 there are plenty of surrounding loops and we need to choose them
 consistently.
-The key feature of the above definition is that the $t$ parameter,
-in addition to allowing to cut out the corrugation
-process in the next chapter, brings a satisfied or refund guarantee,
+The key feature of the above definition is that the $t$ parameter not only
+allows us to cut out the corrugation
+process in the next chapter, but also brings a "satisfied or refund" guarantee,
```
Comment (jcommelin) [inline blueprint/src/loops.tex:227]:
```suggestion
process in the next chapter, but also brings a ``satisfied or refund'' guarantee,
```

Comment (bryangingechen) [reply blueprint/src/loops.tex]: Thanks, I forgot about that! It's been too long since I had to write LaTeX...

File: blueprint/src/loops.tex [hunk 2]
```diff
@@ -14,9 +14,9 @@ \section{Introduction}
   The average of a loop $γ$ is $\bar γ := \int_{𝕊^1} γ(s)\, ds$.
 \end{definition}
 
-In all this document, $E$ and $F$ will denote finite-dimensional
+In this document, $E$ and $F$ will denote finite-dimensional
```
Comment (rwbarton) [inline blueprint/src/loops.tex:17]:
```suggestion
Throughout this document, $E$ and $F$ will denote finite-dimensional
```

**Comment by PatrickMassot** [comment]
Thanks!

**Pull request merged**